create-k8s-chained-sessions.sh

```diff
@@ -15,37 +15,43 @@ declare REGION='us-east-1'
 # Args:
 # 1: parent session id name from leapp
 # 2: sso role name to use for the parent session
-# 3: role name to use for the chained session
+# 3: scope of the IAM role ("panorama" or "eks"). used only for legacy reasons
+#    and can be removed when the keyword is removed from the associated IAM
+#    roles.
+# 4: name of the persona (e.g. admin, dev-writer, etc.) the new session is for
+# 5: version of kubernetes, used to determine the associated IAM role
 function createLeappSession {
     green_echo "creating chained session for $1 with role $3"
```

Comment:
This echo doesn't make sense anymore with the new arguments. It only outputs `panorama` or `eks` instead of a unique role. I think you'll need to do some concatenation to get a role, or just change it to `with persona $4`.

Reply:
Thank you - I went with the persona here since that's the abstraction people will be abstracting with directly anyways, so I guess it puts the idea out there from the start.